package.json

```
@@ -41,16 +41,16 @@
   "dependencies": {
     "@alicloud/pop-core": "^1.7.10",
     "@azure/data-tables": "^13.2.2",
+    "@azure/storage-blob": "^12.14.0",
     "@azure/storage-file-share": "^12.14.0",
     "@azure/storage-queue": "^12.13.0",
-    "@azure/storage-blob": "^12.14.0",
     "@octokit/app": "^3.0.0",
     "@octokit/request": "^3.0.3",
     "@octokit/rest": "^16.3.2",
     "ali-oss": "^6.15.2",
     "argparse": "^2.0.0",
     "async": "^2.6.1",
-    "aws-sdk": "^2.1338.0",
+    "aws-sdk": "^2.1506.0",
```

```suggestion
    "aws-sdk": "^2.1506.0",
```
are we sure that updating the sdk version won't break any existing calls?